akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala

```diff
@@ -676,11 +674,7 @@ class StressMultiJvmNode12 extends StressSpec
 class StressMultiJvmNode13 extends StressSpec
 
 abstract class StressSpec
-    extends MultiNodeSpec({
-      // Aeron media driver must be started before ActorSystem
-      SharedMediaDriverSupport.startMediaDriver(StressMultiJvmSpec)
-      StressMultiJvmSpec
-    })
```

Comment:
ok, strange that this started failing. Did we update Aeron version. `SharedMediaDriverSupport.startMediaDriver` is supposed to start only one.

Reply:
One thing that should be added to `SharedMediaDriverSupport.startMediaDriver` is to only start media driver if the transport is "aeron-udp"

Reply:
Nope we didn't upgrade recently. This has only failed once this way in all of time. I assume a race in isDriverInactive to starting it. I didn't re-create it locally.

Reply:
> One thing that should be added to SharedMediaDriverSupport.startMediaDriver is to only start media driver if the transport is "aeron-udp"

:+1: